python/ray/data/read_api.py

```diff
@@ -1755,6 +1755,86 @@ def create_connection():
     )
 
 
+@PublicAPI(stability="alpha")
+def read_databricks_uc_tables(
+    *,
+    host: str,
+    token: str,
+    warehouse_id: str,
+    catalog: str,
+    schema: str,
+    table_name: Optional[str] = None,
```

Comment:
Can we create a single argument called `table`? If users have already set the current catalog / schema in their notebook environment via `USE CATALOG` / `USE SCHEMA`, can we infer the catalog and schema? If it's too tricky, not a big deal for now

Reply:
we can, but I prefer to make `catalog` and `schema` arguments to be optional, in databricks shard, `catalog` and `schema` already has default values, but with the optional arguments, user can easily override them.